src/primitives/certificate.h

```diff
@@ -74,6 +74,7 @@ class CScCertificate : virtual public CTransactionBase
 
     //CHECK FUNCTIONS
     bool CheckVersionBasic        (CValidationState &state) const override;
+    bool CheckVersionIsStandard   (std::string& reason, const int nHeight) const override;
```

Comment:
probably here 'const' is useless for int var

Reply:
Done